torch/_dynamo/variables/misc.py

```diff
@@ -570,12 +570,16 @@ def call_function(
         if is_original_tensor_torch_function:
             # Instead of tracing inside torch.Tensor.__torch_function__,
             # record the `call_function` or `call_method` call into the graph.
-            from . import TorchVariable
+            from . import ConstantVariable, ConstDictVariable, TorchVariable
 
             original_torch_or_getattr_variable = args[0]
             new_args = args[2].items
-            new_kwargs = args[3].items
-            options = VariableTracker.propagate(self, new_args, new_kwargs.values())
+            if not isinstance(args[3], ConstantVariable):
+                new_kwargs = args[3].items
+                options = VariableTracker.propagate(self, new_args, new_kwargs)
+            else:
+                new_kwargs = ConstDictVariable(dict(), dict).items
```

Comment:
This branch is puzzling to me. You've only checked that args[3] is a ConstantVariable. How do you know that it's empty?! Doesn't seem self evident to me...

Reply:
Honestly, I'm going to remove all of this in the next PR (we can just inline through the base torch function in the next PR), kwargs was None in some of the tests, so I just make items empty in that case.